web.rb

```diff
@@ -57,9 +60,84 @@
     "Charge declined"
   end
 
+end 
+
+#endpoint for Stripe Connect Oauth
+get '/connect' do 
+  #gets two params from Stripe OAUTH
+  scope = params[:scope]
+  auth_code = params[:code]
+
+  p scope
```

Comment:
You probably want to clean up your debug calls if you don't need them anymore, just to keep clutter out of your console. Params often get automatically logged anyway, in production.